modules/applications/pages/ui-builder/faq.adoc

```diff
@@ -132,12 +132,17 @@ Our goal is to make this transition as smooth as possible, minimizing any disrup
 
 
 === Are the interfaces made with Bonita UI Builder compatible with platform mode and self-contained applications ? 
-Yes they are. 
+Yes they are.
```

Comment:
Ponctuation "Yes, they are", and since it is a documentation, would use a complete answer "Yes, they are compatible to both deployment modes"